plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -16,7 +16,6 @@
 
 #include <gio/gio.h>
 #include <glib.h>
-#include <gudev/gudev.h>
```

Comment:
likewise this one too

Reply:
This is slightly tricky; this uses the "changed" signal from udev. I don't really want another vfunc for this, and was pondering a `FuUdevDevice::changed` signal that plugins could connect to.

Reply:
That sounds like a good idea to me.

Reply:
I'll give this a go.

Reply:
I've added the changed signal, but the thunderbolt plugin is somewhat tricky to port as it's got the device-wait stuff too. I'm now thinking we should just finish off https://github.com/hughsie/fwupd/pull/329 and use that and FuUdevDevice together. In the meantime I'd like to merge this PR and get the nvme stuff merged before the next tarball.